src/chttpd/src/chttpd_misc.erl

```diff
@@ -103,6 +103,52 @@ maybe_add_csp_headers(Headers, _) ->
     Headers.
 
 handle_all_dbs_req(#httpd{method='GET'}=Req) ->
+    Params = lists:flatmap(fun({K, V}) -> parse_all_dbs_param(K, V) end,
```

Comment:
Inconsistent formatting of funs. I think most of our codebase uses following formatting.
 ```
lists:flatmap(fun(Element) -> 
   <function body>
end, List),
```

Reply:
Unless whole expression fits the max line length (which is 80 characters).

Reply:
If you change `parse_all_dbs_param(K, V)` to accept tuple you would be able to write:

```
lists:flatmap(fun parse_all_dbs_param/1, chttpd:qs(Req)),
```